### ISSUE TITLE
Everybody,  ByteDance announced a method for animating person by one pic

### ISSUE THREAD

**Issue by shaune0000** [HumanAIGC/AnimateAnyone#21]
https://github.com/magic-research/magic-animate

**Comment by pkuliyi2015** [comment]
It seems to be much worse than their demo?

**Comment by silnt-awaken** [comment]
> It seems to be much worse than their demo?

Right it looks glitchy. This one is still superior. Need demo asap tho

**Comment by yhyu13** [comment]
> I’ve looked into this repo with other devs. It is very hard to support - CN reference only need a large refactor; and the performance is bad for anime, the VRAM consumption might be twice as this work. Not sure if it is worth supporting.

Yep, also notice it use dense pose which do not have face & hand anchor points. This causes very obvious motion artificat for heads and fingers in motion. Which looks not optimal on my local test compare to what this work claims.

PS: Magic animate cost about 8GB VRAM at peak for fp16 inferencing on my local rig
